### PR TITLE
Port to typescrip, added 2 endpoints, added readme

### PR DESCRIPTION
**Pull request by ksyhoo**

### CODE REVIEW

File: socket.yml [hunk 1]
```diff
@@ -7,28 +7,65 @@ keyword:
   - image
   - ImageMagick
 endpoints:
+
   resize:
```
Comment (olapinska) [inline socket.yml]: remove empty line

File: README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+# Syncano Socket for Graphics Magick library
+
+[![Syncano Socket](https://img.shields.io/badge/syncano-socket-blue.svg)](https://syncano.io)
+[![CircleCI branch](https://img.shields.io/circleci/project/github/eyedea-io/syncano-socket-<name>/master.svg)](https://circleci.com/gh/eyedea-io/syncano-socket-<name>/tree/master)
```
Comment (mkucharz) [inline README.md]: Replace all `<name>`.

File: socket.yml [hunk 3]
```diff
@@ -8,27 +8,63 @@ keyword:
   - ImageMagick
 endpoints:
   resize:
-    description: Generate PDF from HTML
+    description: Resize an image
     parameters:
```
Comment (mkucharz) [inline socket.yml]: New version is with `inputs` and `outputs`, please adjust.

File: socket.yml [hunk 4]
```diff
@@ -8,27 +8,63 @@ keyword:
...
+    description: Applay sepia filter to an image
+    parameters:
+      file:
+        type: file
+        description: Image file to applay filter
+        example: test.png
+      success:
+        mimetype: file
+        description: Image filter successfuly applied
+      fail:
```
Comment (mkucharz) [inline socket.yml]: Looks like all here is mixed. Inputs/outputs/parameters etc.

File: socket.yml [hunk 5]
```diff
@@ -8,27 +8,63 @@ keyword:
...
+        mimetype: json
```
Comment (mkucharz) [inline socket.yml]: no such `mimetype`

**Review by mkucharz — CHANGES_REQUESTED** [review]

**Review by mkucharz — CHANGES_REQUESTED** [review]

**Review by mkucharz — APPROVED** [review]

**Pull request merged**